dev_build/client/root/setup.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -ex
+
+# Installing base packages
+yum -y install \
+cron \
+epel-release \
+glibc-static \
+libgomp \
+libstdc++-static \
+perl \
+openssl \
+wget
+
+# Updating ca certs to get epel mirrors to work
+yum -y upgrade ca-certificates --disablerepo=epel
+
+ yum -y groupinstall 'Development Tools'
+
+# Installing condor
+useradd condor
+su -c "wget http://prod-exe.icecube.wisc.edu/htcondor/condor-8.7.2-x86_64_RedHat7-stripped.tar.gz" - condor
+su -c "mkdir ~/condor-8.7.2; cd ~/condor-8.7.2; mkdir local" - condor
+su -c "cd ~/condor-8.7.2; tar -z -x -f ~/condor-8.7.2-*-stripped.tar.gz" - condor
+su -c "cd ~/condor-8.7.2; ./condor-8.7.2-*-stripped/condor_install --local-dir /home/condor/condor-8.7.2/local --make-personal-condor" - condor
+rm -f /home/condor/condor-8.7.2-x86_64_RedHat7-stripped.tar.gz
+chmod 755 /home/condor
+chmod 755 /home/condor/condor-8.7.2/condor.sh
+
+# Installing pyglidein
+useradd pyglidein
+tar xvzf pyglidein.tar.gz
+#TODO: Cleanup how condor handles job outputs
+mkdir /pyglidein/out
+chown -R pyglidein:pyglidein /pyglidein
+#TODO: Cleanup how condor handles job outputs
+chmod 777 /pyglidein
+chmod 777 /pyglidein/out
+yum -y install python-pip
+pip install tornado
+
+# Installing Runit
+wget http://smarden.org/runit/runit-2.1.2.tar.gz
+tar xvzf runit-2.1.2.tar.gz
+cd admin/runit-2.1.2/
+./package/install
+cd /
+rm -f runit-2.1.2.tar.gz
+
+# Installing CVMFS
+yum -y install \
+gawk \
+fuse \
+fuse-libs \
+autofs \
+attr \
+gdb \
+policycoreutils-python
+
+rpm -ivh https://ecsft.cern.ch/dist/cvmfs/cvmfs-config/cvmfs-config-default-1.2-2.noarch.rpm
+rpm -ivh --nodeps https://ecsft.cern.ch/dist/cvmfs/cvmfs-2.2.2/cvmfs-2.2.2-1.el6.x86_64.rpm
```

Comment:
This seems like a pretty old version of cvmfs.  Any reasoning behind this?

Reply:
These commands were from a script in the IceCube Wiki.  Is there a suggested version I should move to?

Reply:
I think 2.3.5 is the latest version.  I hear 2.4 is in beta.